packages/atomic/src/components/atomic-result-children/atomic-result-children.tsx

```diff
@@ -54,30 +72,51 @@ export class AtomicResultChildren {
     }
   }
 
+  private selectInheritedTemplate(result: Result) {
+    const content = this.templatesManager?.selectTemplate(result);
+    if (content) {
```

Comment:
Nitpick: early return `if (!content)` in this method and the next

Reply:
no prob!